lib/TransferManager.cs

```diff
@@ -392,6 +392,7 @@ public static Task DownloadAsync(CloudBlob sourceBlob, string destPath, Download
 
                 BlobRequestOptions requestOptions = Transfer_RequestOptions.DefaultBlobRequestOptions;
                 requestOptions.DisableContentMD5Validation = options.DisableContentMD5Validation;
+                requestOptions.UseTransactionalMD5 = options.UseTransactionalMD5;
```

Comment:
TransactionalMD5 would be enabled for HTTP scenarios, and be disabled for HTTPs scenarios, FYI.

Reply:
Unless the user overrides it, this option will always default to `true`. That is set in `Transfer_RequestOptions.cs`
https://github.com/Azure/azure-storage-net-data-movement/blob/62383ee23aa9df6e4048c3744c8175310a3b3adc/lib/Transfer_RequestOptions.cs#L57-L74

After a quick look I couldn't find any way to reliably determine if a transfer would be HTTPS or HTTP, so although it would be nice to set that default differently for HTTPS, it will have to be up to the user for now.